app/src/main/res/values/strings.xml

```diff
@@ -650,7 +654,7 @@ Please report any problems you encounter via Github.</string>
     <string name="syncthing_terminated">Syncthing was terminated</string>
 
     <!-- Toast shown if syncthing failed to create a config -->
-    <string name="config_create_failed">Failed to create config file</string>
+    <string name="config_create_failed">Failed to create configuration. Consider backing up your sync data, then clear this app\'s data from Android settings.</string>
```

Comment:
Why does failing to create configuration needs backup your data? Also what is sync data? what does that even mean?

Reply:
It's already corrupt, backup of what?

Reply:
I've had existing data in a past fail case when it got corrupt.

Reply:
What existing data? Are you saying a corrupt config deletes data? If the config is corrupt, how does syncthing even know where the data is?

Reply:
Android associates data with apps! Read a book about android.

Reply:
( It's already corrupt, backup of what? ) The Config is corrupt my data lives... yippie, I'll back it up!